lib/javascript/food/foodTable.js

```diff
@@ -0,0 +1,47 @@
+const html = require('./html')
+
+class FoodTable{
+
+  constructor(){}
+
+  renderFoodTable(){
+    $('.new-food-table-container').after(html.foodTable())
+    this.getAllFoods()
+  }
+
+  getAllFoods(){
```

Comment:
same here - this is getting all the foods AND sorting them. any chance we could break this down further?